cmd/generator/cloudspec/spec_modifier.go

```diff
@@ -100,6 +100,19 @@ func Modify(cloudSpec *spec.Swagger) {
 				cloudSpec.Definitions[k].Properties[kk] = prop
 			}
 
+			addExtension := func(model string, field string, key string, value interface{}) {
+				if k == model && kk == field {
+					prop.AddExtension(key, value)
+					cloudSpec.Definitions[k].Properties[kk] = prop
+				}
+			}
+			addExtension("DeploymentGetResponse", "alias", nullableKey, true)
```

Comment:
I'm not saying that this is a good choice but just for the discussion purpose - what if we just change `DeploymentCreateRequest` and `DeploymentUpdateRequest`? In theory, that should be enough to fix the stated issue with `alias` but it should also limit the scope of the breaking change.

Reply:
I agree that it is enough to just update the  *Request models. 

Though one thing I still have to test is the read-update loop in ecctl and how the API actually behaves. So it has to be ensured that the GET response can be used as an update without introducing changes to the alias. It depends a a bit on if the API returns an empty string or null-value if no deployment-alias is set.